### PR TITLE
chore: update readme to reflect new IA

### PR DESCRIPTION
**Pull request by katiegoines**
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -58,19 +92,21 @@ When editing content that hasn't been migrated, you may see the following patter
...
-This fragment would exist in: `pages/src/fragments/lib/datastore/js/conflict.mdx`
+This fragment would exist in: `src/fragments/lib/datastore/js/conflict.mdx`
 
 ### Inline Filters
 
 We are incorporating the use of `<InlineFilters>` to add platform-specific content within the context of one page rather than in fragments. These filters allow you to still specify content by platform and they reference platforms using the same naming convention as our fragments. You can enclose your platform-specific content by updating the opening tag:
 
 ```md
-<InlineFilter filters={["javascript", "react-native", "android", "swift", "flutter"]}>
+<InlineFilter filters={["javascript", "react-native", "angular", "nextjs", "react", "vue"]}>
```
Comment (heatheramz) [inline Readme.md:109]: Before we showed all options - do we still want to show them like we do in the header example above and just reiterate that they edit out what they aren't using.
```suggestion
<InlineFilter filters={["javascript", "react-native", "angular", "nextjs", "react", "vue", "android", "swift", "flutter"]}>
```

**Review by heatheramz — COMMENTED** [review]
One Q on this one

Also did we want to add something about linking pages that should have [platform] in the URL path and/or specify full paths from under the pages folders eg
/[platform]/build-a-backend/auth/manage-mfa/ not ../manage-mfa/

**Comment by hibler13** [comment]
@katiegoines Just finished my edit.

File: Readme.md [hunk 2]
```diff
@@ -129,6 +165,10 @@ let mut a = String::from("a");
 </BlockSwitcher>
 ````
 
+### Internal links
```
Comment (katiegoines) [inline Readme.md:168]: @heatheramz - Good callout - does this cover it?

**Review by hibler13 — APPROVED** [review]
LGTM

**Pull request merged**